instrumentation/jms-1.1/src/latestDepTest/groovy/JMS2Test.groovy

```diff
@@ -45,6 +46,13 @@ import static io.opentelemetry.trace.Span.Kind.CONSUMER
 import static io.opentelemetry.trace.Span.Kind.PRODUCER
 
 class JMS2Test extends AgentTestRunner {
+  static {
+    ConfigUtils.updateConfig {
+      System.setProperty("ota.trace.classes.exclude", "org.springframework.jms.config.JmsListenerEndpointRegistry\$AggregatingCallback,org.springframework.context.support.DefaultLifecycleProcessor\$1")
```

Comment:
This seems surprising. I thought `ota.trace.classes.exclude` only applied to annotation instrumentation?

Reply:
It is used in `io.opentelemetry.auto.tooling.AgentInstaller#matchesConfiguredExcludes` so applies to everything.

Reply:
Oh, I didn't realize that. Still curious why this is needed? If related to your [gitter question](https://gitter.im/open-telemetry/opentelemetry-java-instrumentation?at=5f155d76a874ff08d288cdb5), maybe these classes need to be added to `AdditionalLibraryIgnoresMatcher`?

Reply:
Btw, I also opened #737 to consider applying `trace.classes.exclude` only to annotation instrumentation

Reply:
I will try to formulate my problem and will open an issue regarding `AdditionalLibraryIgnoresMatcher`